test/browser_tests/page_objects/page_dbwu-small-businesses.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+function SmallBusinessess() {
+
+  this.get = function() {
+    browser.get( 'doing-business-with-us/small-businesses/' );
+  };
+
+  this.pageTitle = browser.getTitle;
+
+  this.sideNav = element( by.css( '.nav-secondary' ) );
+
+  this.mainTitle = element( by.css( '[data-qa-hook="main-title"]' ) );
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/aa3d436b18af4069fff661b4b51d0fa772a64846